core/attributedef.class.inc.php

```diff
@@ -1744,6 +1744,15 @@ public function GetHasConstraint()
 		return $this->GetOptional('with_php_constraint', false);
 	}
 
+	/**
+	 * @return bool true if Attribute has computation (DB_LINKS_CHANGED event propagation, `with_php_computation` attribute xml property), false otherwise
+	 * @since 3.1.1 3.2.0 N°6228
+	 */
+	public function GetHasComputation()
```

Comment:
Double action in the name
```suggestion
	public function HasComputation()
```